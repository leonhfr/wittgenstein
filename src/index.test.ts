import * as index from './index';

// Code.
describe('index', () => {
  it('should expose the expected entries', () => {
    expect(index).toMatchSnapshot();
  });
});
