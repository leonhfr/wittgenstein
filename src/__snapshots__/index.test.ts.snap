// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should expose the expected entries 1`] = `
Object {
  "Area": [Function],
  "Photo": [Function],
}
`;
